python/neuroglancer/pipeline/storage.py

```diff
@@ -214,30 +342,34 @@ def get_path_to_file(self, file_path):
                              file_path])
         return  os.path.join(*clean)
 
-    def put_file(self, file_path, content, compress):
+    def put_file(self, file_path, content, content_type, compress):
```

Comment:
This is possible, but is more complicated and comes with its own set of issues. Let's do the simple thing until we need it.